BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeApiConfiguration.kt

```diff
@@ -0,0 +1,30 @@
+package com.braintreepayments.api
+
+import android.text.TextUtils
+import org.json.JSONObject
+
+/**
+ * Contains the remote Braintree API Configuration for the Braintree SDK.
+ * @property accessToken The Access Token for Braintree API.
+ * @property url the base url for accessing Braintree API.
+ */
+internal data class BraintreeApiConfiguration(
+    val accessToken: String = "",
+    val url: String = ""
```

Comment:
Why default to empty string, when the `optString()` is already falling back to empty string?

Reply:
That's a good call. I'll remove these default assignments.